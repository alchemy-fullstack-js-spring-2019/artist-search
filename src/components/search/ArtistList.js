import React from 'react';
import PropTypes from 'prop-types';
import Artist from './Artist';

function ArtistList({ artistArr }) {
  const allArtists = artistArr.map(artist => {
    return (
      <li key={artist.id}>
        <Artist artist={artist.name} id={artist.id} />
      </li>
    );
  });

  return <ul>{allArtists}</ul>;
}

ArtistList.propTypes = {
  artistArr: PropTypes.array.isRequired
};

export default ArtistList;
