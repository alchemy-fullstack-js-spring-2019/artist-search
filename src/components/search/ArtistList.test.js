import React from 'react';
import { shallow } from 'enzyme';
import ArtistList from './ArtistList';

describe('ArtistList component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ArtistList artistArr={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
