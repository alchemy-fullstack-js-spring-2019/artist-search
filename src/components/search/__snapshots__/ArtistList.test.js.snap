// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistList component matches snapshot 1`] = `<ul />`;
